apps/api/src/hyp3_api/validation.py

```diff
@@ -70,31 +70,38 @@ def check_granules_exist(granules, granule_metadata):
         raise GranuleValidationError(f'Some requested scenes could not be found: {", ".join(not_found_granules)}')
 
 
-def check_dem_coverage(granule_metadata):
+def check_dem_coverage(_, granule_metadata):
```

Comment:
```suggestion
def check_dem_coverage(job, granule_metadata):
```
I couldn't decide which of these is better. I ended up going with `_` because it communicates that the variable is unused.